diversity_in_cinema/stats.py

```diff
@@ -0,0 +1,127 @@
+import pandas as pd
+import numpy as np
+from functools import reduce
+
+
+def movie_stats(df_movie_location):
+    '''
+    input: .csv file containing 5 columns with the following information of a movie: unnamed:0, frame_number, face_id, gender and race
+    output: 2 pandas dataframe with some statistics (data frame 1: gender statistics, data frame 2: race statistics)
+     '''
+    # Import the movie´s data (csv-> pandas dataframe)
+    df_movie = pd.read_csv(df_movie_location)
+
+    # clean up of unnecessary columns
+    df_movie = df_movie.drop(columns=['Unnamed: 0'])
```

Comment:
This 'Unnamed: 0' column will not be in the final input dataframe so we don't need to remove it.